.github/workflows/app-build.yml

```diff
@@ -0,0 +1,36 @@
+name: App Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+
+      - name: Build Unsigned Release
+        run: ./gradlew --no-daemon --info assembleRelease
+
+      - name: Split APK release types
+        run: |
+          mkdir -p build/jellyfin-publish/release-libre build/jellyfin-publish/release-proprietary;
+          mv app/build/outputs/apk/*/*/jellyfin-android-*-libre-release-unsigned.apk build/jellyfin-publish/release-libre/;
+          mv app/build/outputs/apk/*/*/jellyfin-android-*-proprietary-release-unsigned.apk build/jellyfin-publish/release-proprietary/;
```

Comment:
We need the debug versions here or just all .apk files. An unsigned release can't be installed on a device.

Reply:
would that not require a "debug-keystore" to be passed into the build step of the `app-build` workflow?